env/staging/kustomization.yaml

```diff
@@ -26,8 +29,9 @@ configMapGenerator:
   env: .env
 
 patches:
- - performance.yaml
- - services.yaml
+  - celery-deployment.yaml
```

Comment:
Hopefully this will mean that the copy of celery-deployment.yaml in the staging env overrides the one in base.